functions-and-operators/miscellaneous-functions.md

```diff
@@ -23,15 +24,386 @@ TiDB supports most of the [miscellaneous functions](https://dev.mysql.com/doc/re
+[ [`IS_UUID()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_is-uuid)                  | Whether argument is an UUID                       |
 | [`NAME_CONST()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_name-const)            | Can be used to rename a column name               |
 | [`SLEEP()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_sleep)                      | Sleep for a number of seconds. Note that for [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters, the `SLEEP()` function has a limitation wherein it can only support a maximum sleep time of 300 seconds.       |
 | [`UUID()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_uuid)                        | Return a Universal Unique Identifier (UUID)       |
 | [`UUID_TO_BIN()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_uuid-to-bin)          | Convert UUID from text format to binary format    |
 | [`VALUES()`](https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_values)                    | Defines the values to be used during an INSERT    |
 
+### ANY_VALUE()
+
+```
+mysql> CREATE TABLE fruits (id INT PRIMARY KEY, name VARCHAR(255));
+Query OK, 0 rows affected (0.14 sec)
+
+mysql> INSERT INTO fruits VALUES (1,'apple'),(2,'apple'),(3,'pear'),(4,'banana'),(5, 'pineapple');
+Query OK, 5 rows affected (0.01 sec)
+Records: 5  Duplicates: 0  Warnings: 0
+
+mysql> SELECT id,name FROM fruits GROUP BY name;
+ERROR 1055 (42000): Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'test.fruits.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
+mysql> SELECT ANY_VALUE(id),GROUP_CONCAT(id),name FROM fruits GROUP BY name;
++---------------+------------------+-----------+
+| ANY_VALUE(id) | GROUP_CONCAT(id) | name      |
++---------------+------------------+-----------+
+|             1 | 1,2              | apple     |
+|             3 | 3                | pear      |
+|             4 | 4                | banana    |
+|             5 | 5                | pineapple |
++---------------+------------------+-----------+
+4 rows in set (0.00 sec)
+```
+
+Here TiDB returns an error for the first `SELECT` query as the `id` column is nonaggregated. For the second query we use the `ANY_VALUE()` function to get one value out of the group and `GROUP_CONCAT()` to get all values of the group. This can be used to avoid having to change the SQL mode for nonaggregated columns.
+
+### BIN_TO_UUID()
+
+`BIN_TO_UUID()` and `UUID_TO_BIN()` can be used to convert between a textual format UUID to a binary format. The second argument for these functions depends the ordering of the fields in the binary format.
```

Comment:
Maybe add a bit more content on why this is a good idea, like:
- saves space
- more control on how to compare and sort UUIDs, like if one want to avoid hot spots by sorting first on time, or to group values by the same client for faster retrieval.
Maybe also provide more info about UUID version or a link somewhere to read more about it.

Or just `See UUID()`?